nibe/data/f750.json

```diff
@@ -6037,13 +6101,12 @@
     "write": true
   },
   "47340": {
+    "title": "Cooling with room sensor",
+    "name": "cooling-with-room-sensor-47340",
+    "size": "s16",
     "factor": 1,
-    "max": 18.0,
     "mappings": {
-      "0": "OFF",
-      "1": "ON",
-      "2": "BT50",
-      "10": "RMU-BT50"
+      "-32768": "--"
```

Comment:
This value won't ever be mapped. 80 00 in hex will be converted to None for size:  s16

Reply:
I just added it based on the uplink API response, I can't see the correct size there thou.
The important stuff is to add name, title and size since Home Assistant throws an warning about it missing
`2023-11-01 05:57:33.585 WARNING (MainThread) [nibe.nibe.heatpump] Failed to register coil 47340: Coil.__init__() missing 3 required positional arguments: 'name', 'title', and 'size'`

Should I remove the mapping, change the size or just keep it as is?